README.md

```diff
@@ -397,13 +397,23 @@ Here is an example with `ngx_lua`:
          local result = answ["result"]
 
          if result ~= nil then
-           ngx.status = result[1]
-           for k, v in pairs(result[2]) do
-             ngx.header[k] = v
-           end
-
-           table.remove(result, 1)
-           ngx.say(cjson.encode(result))
+                ngx.status = result[1]
+                for k, v in pairs(result[2]) do
+                    ngx.header[k] = v 
+                end 
+
+                local body = result[3]
+                if type(body) == "string" then
+                    ngx.header["content_type"] = "text/plain"
+                    ngx.print(body)
+                elseif type(body) == "table" then
+                    -- we have not empty objects in our APIs
+                    local body = cjson.encode(body):gsub("{}", "[]")

```

Comment:
I'll remove `:gsub("{}", "[]")`, because it is a workaround we have in a project specifically for our API implementation. If you're using OpenResty and want to achieve the same behaviour it is better to use https://github.com/openresty/lua-cjson/tree/9931667547e2eda9911ec07b0f408b1a04b8a46c#encode_empty_table_as_object . Anyway, I don't think it should be a part of the common example.